ui/app/rif/actions/index.js

```diff
@@ -248,6 +249,20 @@ function deletePendingChainAddress (chain, isSubdomain) {
   }
 }
 
+function deletePendingSubdomain (domainName, subdomain) {
+  return (dispatch) => {
```

Comment:
I dont understand this, why deletePendingSubdomain is a rns.register invocation? Isnt this just remove from a list when is confirmed?

Reply:
Also, why is a promise?

Reply:
This is a rns.register invocation because there's where we build all the subdomain logic (So there's where is the pending array)